src/iotEdgeExplorer.ts

```diff
@@ -102,6 +114,23 @@ export class IoTEdgeExplorer extends BaseExplorer {
         }
     }
 
+    private async isValidDeploymentJson(json: string): Promise<boolean> {
+        try {
+            const schema = (await axios.get(Constants.DeploymentJsonSchemaUrl)).data;
+            const ajv = new Ajv({ allErrors: true });
+            const valid = ajv.validate(schema, JSON.parse(json));
+            if (!valid) {
+                this._outputChannel.show();
+                this.outputLine(Constants.IoTHubEdgeLabel, `There are errors in deployment json file:\n${ajv.errorsText(null, { separator: ",\n" })}`);

```

Comment:
when the schema is invalid, we need a more clear way to let user know. So message box here makes sense.

Reply:
Use message box instead